src/runtime/virtcontainers/fs_share_linux.go

```diff
@@ -266,8 +280,46 @@ func (f *FilesystemShare) ShareFile(ctx context.Context, c *Container, m *Mount)
 			}
 
 			dstPath := filepath.Join(guestPath, srcPath[len(srcRoot):])
+			f.Logger().Infof("ShareFile: Copying file from src (%s) and dest (%s)", srcPath, dstPath)
```

Comment:
`src (%s) and dest (%s)`  -> `src (%s) to dest (%s)`

Reply:
Done